apps/manga-reader/web/src/main.tsx

```diff
@@ -1,13 +1,25 @@
 import "./index.css";
 
+import { ClerkLoaded, ClerkProvider } from "@clerk/clerk-react";
 import { RouterProvider } from "@tanstack/react-router";
 import React from "react";
 import ReactDOM from "react-dom/client";
 
-import { router } from "./routes/rooter";
+import { router } from "./routes/router";
+
+const clerkPubKey = import.meta.env["VITE_REACT_APP_CLERK_PUBLISHABLE_KEY"];
+
+// eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+if (!clerkPubKey) {
+  throw new Error("Missing Publishable Key");
+}
```

Comment:
**Suggestion:** Consider wrapping the logic of checking and throwing an error if the clerkPubKey is not present into a separate function for better readability and reusability.
```suggestion
function checkClerkPubKey() {
  const clerkPubKey = import.meta.env["VITE_REACT_APP_CLERK_PUBLISHABLE_KEY"];
  if (!clerkPubKey) {
    throw new Error("Missing Publishable Key");
  }
  return clerkPubKey;
}
const clerkPubKey = checkClerkPubKey();
```